libportal/remote.h

```diff
@@ -24,6 +24,25 @@
 
 G_BEGIN_DECLS
 
+#define XDP_TYPE_SCREENCAST_SESSION (xdp_screencast_session_get_type ())
+
+XDP_PUBLIC
+G_DECLARE_DERIVABLE_TYPE (XdpScreenCastSession, xdp_screencast_session, XDP, SCREENCAST_SESSION, XdpSession)
+
+struct _XdpScreenCastSessionClass
+{
+  GObjectClass parent_class;
+
+  void (*close) (XdpSession *session);
+
+  gpointer padding[24];
+};
```

Comment:
Seems like this needs to go to a `remote-private.h` header too? We don't want client application code deriving from it

Reply:
just ftr, this is resolved, it just required the `XdpRemoteDesktopSession` to be a private implementation too.